### PR TITLE
Fixed the MySQL on CentOS blueprint

### PR DESCRIPTION
**Pull request by cb-ishu**
# Development Prerequisites
- **Developer:** @cb-ishu
- **Partner:** 
- **Jira Story:** [DEV-18563](https://cloudbolt.atlassian.net/browse/DEV-18563)
- **Design Doc:** N/A

## Summary of Proposed Changes
* Fixed issues where ordering blueprint for MySQL on CentOS.
#### Current status
* Blueprint is getting ordered successfully.
#### NOTES
* Uncomment lines 19-23 if space issues exist.
* Attached blueprint zip with proposed changes such as regex, and description for database_root_password parameter on the order form.
[mysql_on_centos (1).zip](https://github.com/CloudBoltSoftware/cloudbolt-forge/files/8141686/mysql_on_centos.1.zip)

### CODE REVIEW

File: blueprints/vmware/mysql/install_mysql_on_centos.sh [hunk 1]
```diff
@@ -1,34 +1,79 @@
...
+echo ' -> Importing GPG keys online'
+rpm --import https://repo.mysql.com/RPM-GPG-KEY-mysql-2022
+echo ' -> Installing mysql server (community edition)'
+yum install -y mysql-server
 
-if [ "$var" -eq 1 ];then
-    yum update curl -y
+# commands to create a swap parition for a smaller sized instance
+# echo ' -> Adding swap partition for fixinf memory issues'
+# dd if=/dev/zero of=/swapfile bs=1M count=1024
```
Comment (yogeshmhaskule93) [inline blueprints/vmware/mysql/install_mysql_on_centos.sh:19]: @cb-ishu  Can you remove commented code, if not required.

File: blueprints/vmware/mysql/install_mysql_on_centos.sh [hunk 2]
```diff
@@ -1,34 +1,79 @@
...
+# # free -k
+# # swapon -a
+# # swapoff -a
 
-    curl -O -1 -L https://dev.mysql.com/get/mysql80-community-release-el6-3.noarch.rpm
-    yum localinstall mysql80-community-release-el6-3.noarch.rpm -y
-    yum install mysql-community-server -y
-    service mysqld start
+echo ' -> adding configuration to my.cnf file to access mysql without default password'
+echo '
```
Comment (yogeshmhaskule93) [inline blueprints/vmware/mysql/install_mysql_on_centos.sh:30]: Is this statement required (Or it would have meaningful log message)

**Review by yogeshmhaskule93 — CHANGES_REQUESTED** [review]

File: blueprints/vmware/mysql/install_mysql_on_centos.sh (hunk 1 above)
Comment (cb-ishu) [reply blueprints/vmware/mysql/install_mysql_on_centos.sh:19]: @yogeshmhaskule93 I can surely remove these lines, but these are important if the testing is to be done on smaller instances. These lines of code aren't required if the instances will always have sufficient memory. Let me know if I should remove these.

File: blueprints/vmware/mysql/install_mysql_on_centos.sh (hunk 2 above)
Comment (cb-ishu) [reply blueprints/vmware/mysql/install_mysql_on_centos.sh:30]: @yogeshmhaskule93 this statement is not required, they identify the steps of execution but are only required for tracking the progress of the script. I will remove these echo statements.

Comment (cb-ishu) [reply blueprints/vmware/mysql/install_mysql_on_centos.sh:14]: Updated the script, please check.

**Review by yogeshmhaskule93 — APPROVED** [review]
:+1:

**Comment by yogeshmhaskule93** [comment]
Closing pull request without merge as newer changes approved and tested in a new PR.

**Pull request closed without merging**